contracts/VetoGuard.sol

```diff
@@ -21,43 +22,26 @@ contract VetoGuard is
     IGnosisSafe public gnosisSafe;
     mapping(bytes32 => uint256) transactionQueuedBlock;
 
-    constructor(
-        address _owner,
-        uint256 _executionDelayBlocks,
-        address _vetoERC20Voting,
-        address _gnosisSafe
-    ) {
-        bytes memory initializeParams = abi.encode(
-            _executionDelayBlocks,
-            _owner,
-            _vetoERC20Voting,
-            _gnosisSafe
-        );
-        setUp(initializeParams);
-    }
-
```

Comment:
Why has the constructor been removed?

Reply:
Having the constructor creates a circular dependency for the gnosis safe. More importantly, it sets the delay blocks which will cause issues for other txs. Also - I hate constructors.

Reply:
Ok, I'm cool with removing it

Reply:
Now that I am thinking about it - We could add this back and just have the gnosis safe deploy this, that way it has access to the gnosis safe msg.sender. Actually - it will not since msg.sender will be the moduleFactory address.